mava/components/jax/updating/parameter_server.py

```diff
@@ -29,7 +29,7 @@
 @dataclass
 class ParameterServerConfig:
     checkpoint: bool = True
-    checkpoint_subpath: str = "~/mava/"
+    experiment_path: str = "~/mava/"
```

Comment:
I'm open to suggestions on this, but I think changing this to be a more general name is useful

Reply:
I agree with this. The comment we have everwhere `# Checkpointer appends "Checkpoints" to checkpoint_dir` is not ideal but this is because we are using Acme's checkpointer

Reply:
But you would need to update all other run files